btcutil/gcs/builder/builder.go

```diff
@@ -369,3 +369,19 @@ func MakeHeaderForFilter(filter *gcs.Filter, prevHeader chainhash.Hash) (chainha
 	// above.
 	return chainhash.DoubleHashH(filterTip), nil
 }
+
+// MakeHeaderForUtreexoCFilter makes a filter chain header for a utreexoc filter, given the
+// filter data and the previous filter chain header.
+func MakeHeaderForUtreexoCFilter(filterData []byte, prevHeader chainhash.Hash) (chainhash.Hash, error) {
+	filterTip := make([]byte, 2*chainhash.HashSize)
+	filterHash := chainhash.DoubleHashH(filterData)
```

Comment:
Probably ok with just a single hash? But this needs to be coordinated with https://github.com/Davidson-Souza/Floresta/

@Davidson-Souza

Reply:
We need to comply with BIP0157, so double:

> The canonical hash of a block filter is the double-SHA256 of the serialized filter.

Reply:
> Probably ok with just a single hash? But this needs to be coordinated with https://github.com/Davidson-Souza/Floresta/
> 
> @Davidson-Souza

I think the double Hash is used throughout the codebase and as per BIP157 too, So I guess it's okay as is?